app/src/main/res/layout/activity_bill.xml

```diff
@@ -136,25 +103,53 @@
                                 android:layout_width="wrap_content"
                                 android:layout_height="wrap_content"
                                 android:layout_gravity="center_vertical"
-                                app:srcCompat="@drawable/ic_phone_receiver_24dp"
-                                tools:ignore="ContentDescription"/>
+                                app:srcCompat="@drawable/ic_short_hair_24dp"
+                                tools:ignore="ContentDescription"
+                                />
 
                             <android.support.design.widget.TextInputLayout
                                 android:layout_width="match_parent"
                                 android:layout_height="wrap_content"
                                 android:paddingEnd="@dimen/dp_8"
                                 android:paddingStart="@dimen/dp_8"
-                                bind:errorText="@{viewModel.customerPhoneError}">
+                                bind:errorText="@{viewModel.customerNameError}"
+                                >
 
                                 <EditText
                                     android:layout_width="match_parent"
                                     android:layout_height="wrap_content"
-                                    android:hint="@string/title_customer_phone"
-                                    android:inputType="number"
-                                    android:text="@={viewModel.billRequest.phone}"
+                                    android:hint="@string/title_customer_name"
+                                    android:inputType="textPersonName"
+                                    android:text="@={viewModel.billRequest.customerName}"
                                     android:textSize="@dimen/sp_12"/>
                             </android.support.design.widget.TextInputLayout>
                         </LinearLayout>
+
+                        <LinearLayout
```

Comment:
WARNING: [UseCompoundDrawables] This tag and its children can be replaced by one `<TextView/>` and a compound drawable
 A `LinearLayout` which contains an `ImageView` and a `TextView` can be more efficiently handled as a compound drawable (a single TextView, using the `drawableTop`, `drawableLeft`, `drawableRight` and/or `drawableBottom` attributes to draw one or more images adjacent to the text).

If the two widgets are offset from each other with margins, this can be replaced with a `drawablePadding` attribute.

There's a lint quickfix to perform this conversion in the Eclipse plugin.